checker/check-request-property-pattern-added-or-changed_test.go

```diff
@@ -0,0 +1,79 @@
+package checker_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tufin/oasdiff/checker"
+	"github.com/tufin/oasdiff/diff"
+)
+
+// CL: changing request property pattern
+func TestRequestPropertyPatternChanged(t *testing.T) {
+	s1, err := open("../data/checker/request_property_pattern_added_or_changed_base.yaml")
+	require.NoError(t, err)
+	s2, err := open("../data/checker/request_property_pattern_added_or_changed_revision.yaml")
+	require.NoError(t, err)
+
+	s2.Spec.Paths["/test"].Post.RequestBody.Value.Content["application/json"].Schema.Value.Properties["name"].Value.Pattern = "^[\\w\\s]+$"
+
+	d, osm, err := diff.GetWithOperationsSourcesMap(getConfig(), s1, s2)
+	require.NoError(t, err)
+	errs := checker.CheckBackwardCompatibilityUntilLevel(singleCheckConfig(checker.RequestPropertyPatternUpdatedCheck), d, osm, checker.INFO)
+	require.Len(t, errs, 1)
+	require.Equal(t, checker.ApiChange{
+		Id:          "request-property-pattern-changed",
+		Text:        "changed the pattern for the request property 'name' from '^\\w+$' to '^[\\w\\s]+$'",
```

Comment:
Please change to:
changed the pattern of the request property 'name' from '^\\w+$' to '^[\\w\\s]+$'

Reply:
need an extra '\' to avoid compilation errors